test/Rings/solving.jl

```diff
@@ -40,11 +40,10 @@
     @test res[2].denom == 2*x
 
     # issue 1743
-    # disabled until fixed in msolve
-    #= R, (x1, x2) = polynomial_ring(QQ, ["x1", "x2"])
-     = I = ideal(R, [x1 + ZZRingElem(2)^100, x2 + ZZRingElem(2)^100])
-     = sols = Vector{QQFieldElem}[[-1267650600228229401496703205376, -1267650600228229401496703205376]]
-     = @test sols == real_solutions(I)[1] =#
+    R, (x1, x2) = polynomial_ring(QQ, ["x1", "x2"])
+    I = ideal(R, [x1 + ZZRingElem(2)^100, x2 + ZZRingElem(2)^100])
```

Comment:
Completely optional, but I'd consider this more idiomatic (but it generates identical code, so feel free to ignore)
```suggestion
    I = ideal(R, [x1 + ZZ(2)^100, x2 + ZZ(2)^100])
```